ui/app/services/version.js

```diff
@@ -89,6 +95,21 @@ export default class VersionService extends Service {
     }
   }
 
+  @keepLatestTask
+  *getActivatedFeatures() {
+    // Response could change between user sessions so fire off endpoint without checking if activated features are already set.
+    try {
+      const response = yield this.store
+        .adapterFor('application')
+        .ajax('/v1/sys/activation-flags', 'GET', { unauthenticated: true, namespace: null });
+      this.activatedFeatures = response.data?.activated;
+      return;
+    } catch (error) {
+      if (DEBUG) console.error(error); // eslint-disable-line no-console
+      return [];
```

Comment:
Let me double check this because I think not returning an empty array causes a failure on the `this.activatedFeatures.includes()`, when instead we should be surfacing the error on the template. Again, let me double check.

Reply:
Double checked, doesn't seem to be an issue to remove.